### DISCUSSION TITLE
Opinionated Formatter for Power Fx

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/Power-Fx#2374] [Ideas]
Power Fx expressions/nesting can get quite deep. It would be good to have an opinionated code formatter for Power Fx along the lines of Prettier and Csharpier.

Should the formatting apply to the serialised or user friendly representation of Power Fx?

Thoughts?

**Comment by MikeStall** [comment]
Power Apps does have a builtin formatter in their editor. What do you think of that one?
